### ISSUE TITLE
[CHANGED] Set manufactureLimit to 10000 from 1000. This allows for much larger factory crate stack sizes

### ISSUE THREAD

**Issue by Spartan5150** [TarkinII/Core3#3]
Still spits out the "Only 1-1000" for size, tried this on a few different items. No dice.

https://i.gyazo.com/aed96c6e1d0c41ddeeeef5fc0e07b59d.png

**Comment by kinshi** [comment]
The client will always display 1000 despite this change and the slider will only go up to 1000. The desired count needs to be manually entered in the text entry field.

Please verify that counts > 1000 are manually entered

**Comment by Spartan5150** [comment]
I have tested and verified that counts do not go to 10K when manually entered, it spits it back out to me with no schem.

**Comment by stacy19201325** [comment]
I don't know if there's a way to attach an issue to a commit, but it's this one: https://github.com/TarkinII/Core3/commit/82cd237e3ce4e331721237b5bf39ea02f9a47026

**Comment by stacy19201325** [comment]
Confirmed not working, I created a manufacturing schematic of 1000, thinking this should get me up to 10,000 items if I put enough resources in.  I put enough resources to make 2,000 items, but once it hit 1000, the manufacturing stopped and the manufacturing schematic disappeared.

**Comment by Tatwi** [comment]
When I did this initially on Tarkin 1, I had to fudge the C++ function so that when you set 1,000 on the client, the function on server that makes schematic uses the number 10,000 instead of 1,000. This is because the client only sends 1 to 1,000 and I have no idea how one could change the client to send a different value.

https://github.com/TarkinII/Core3/blob/82cd237e3ce4e331721237b5bf39ea02f9a47026/MMOCoreORB/src/server/zone/objects/manufactureschematic/ManufactureSchematic.idl

public int getManufactureLimit() {
    if (manufactureLimit == 1000)
       return 10000; 

   return manufactureLimit;
}

That would over-ride the 1,000 and make it 10,000 instead.
